amcl/src/amcl_node.cpp

```diff
@@ -1359,6 +1497,7 @@ AmclNode::laserReceived(const sensor_msgs::LaserScanConstPtr& laser_scan)
       if(d > particlecloud_pub_interval_)
       {
         particlecloud_pub_.publish(cloud_msg);
+        // debug_particlecloud_pub_.publish(cloud_msg);
```

Comment:
I think it would be better to uncomment. Would there be any problems if I unsubscribe? Load issues, other logic malfunctioning, etc.?

コメントを解除する方が良いと思います。解除すると何か問題になりそうですか?負荷の問題や、他のロジックが誤動作するとか。

Reply:
案2の実装を見込んでdebug用に用意していたものです。削除しました。
done in 7842237711bc996cf51cb25e7c52365116598600